{{cookiecutter.github_repository}}/provisioner/roles/common/tasks/main.yml

```diff
@@ -15,6 +15,9 @@
 #  when: vm == 1
 
 # Ansible requires aptitude and python2 for various tasks
+- name: Set hostname
+  action: shell hostnamectl set-hostname {{ domain_name }}
```

Comment:
So currently, the a single machine handles a single host. If and when ansible made to handle different host, we can have an explicit `hostname` variable, until then i don't feel like adding another variable to the stack. Thoughts?